client/extensions/woocommerce/woocommerce-services/state/shipping-label/actions.js

```diff
@@ -531,6 +532,20 @@ export const confirmPackages = ( orderId, siteId ) => ( dispatch, getState ) =>
 	tryGetLabelRates( orderId, siteId, dispatch, getState );
 };
 
+export const saveCustoms = ( orderId, siteId ) => {
```

Comment:
`saveCustoms` doesn't get used outside of this file, so no need to export.

Reply:
Done https://github.com/Automattic/wp-calypso/pull/24975/commits/e658a9d562b509fd9067b8a6f8123758f7e00e70